From 42240dcbbce8ec744401d4a464f71c3fbd3fca1b Mon Sep 17 00:00:00 2001
From: Emil Velikov <emil.l.velikov@gmail.com>
Date: Mon, 9 Mar 2015 11:57:40 +0000
Subject: [PATCH 4/9] autogen.sh: pass --force to autoreconf, quote string
 variables

By passing --force autoreconf will update all the aux files, which would
otherwise be ignored if one updates autoconf/automake.

Quote the $srcdir, $ORIGDIR and $0 variables to prevent fall-outs, when
it contains space.

v2: Also handle $srcdir and $0.

Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
---
 autogen.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index fc34bd5..81e182d 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -6,8 +6,8 @@ test -z "$srcdir" && srcdir=.
 ORIGDIR=`pwd`
 cd $srcdir
 
-autoreconf -v --install || exit 1
-cd $ORIGDIR || exit $?
+autoreconf --force --verbose --install || exit 1
+cd "$ORIGDIR" || exit $?
 
 if test -z "$NOCONFIGURE"; then
     $srcdir/configure "$@"
-- 
2.8.3

